recipes/libdispatch/all/conanfile.py

```diff
@@ -58,6 +55,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.libs = tools.collect_libs(self)
         if self.settings.os == "Linux":
+            self.cpp_info.libs = ["dispatch", "BlocksRuntime"]
             self.cpp_info.system_libs = ["pthread"]
+        else:
+            self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
You can fix `package_info()` for all platforms: https://github.com/apple/swift-corelibs-libdispatch/blob/f13ea5dcc055e5d2d7c02e90d8c9907ca9dc72e1/src/CMakeLists.txt#L135

Reply:
I can not understand your point.
You are telling me to remove 

`self.cpp_info.libs = tools.collect_libs(self)` for `self.settings.os != "Linux"`

Reply:
Just to be explicit for all platforms.
Non Darwin: BlocksRuntime is created and installed
Windows: dispatch depends on several system libs
Linux: dispatch seems to depend on librt also

You can also use conan components, because libdispatch installs CMake config file dispatch-config.cmake

Reply:
I do not understand what you mean by "conan components".

Reply:
This: https://docs.conan.io/en/latest/creating_packages/package_information.html#using-components

Reply:
> Linux: dispatch seems to depend on librt also https://github.com/apple/swift-corelibs-libdispatch/blob/f13ea5dcc055e5d2d7c02e90d8c9907ca9dc72e1/src/CMakeLists.txt#L135

@SpaceIm 
The linking to `librt` is conditional to `LibRT_FOUND`.
Do you know how to express that in the Conan recipe?

Reply:
We can't, usually we add `rt` in `cpp_info.system libs` if os = Linux in `package_info()`.

Reply:
Done, thank you!